tracee-rules/engine/engine.go

```diff
@@ -110,14 +110,14 @@ func signatureStart(signature types.Signature, c chan types.Event, wg *sync.Wait
 // it runs continuously until stopped by the done channel
 // once done, it cleans all internal resources, which means the engine is not reusable
 // note that the input and output channels are created by the consumer and therefore are not closed
-func (engine *Engine) Start(done chan bool) {
+func (engine *Engine) Start(parsedEvents bool, done chan bool) {
```

Comment:
With parsedEvents option passed to NewEngine and stored as Engine's field we could avoid changing multiple method signatures.

Reply:
fixed